app/views/feedback.py

```diff
@@ -0,0 +1,93 @@
+from flask import Blueprint, redirect, url_for, escape, request, current_app, g
+from flask_login import login_required, current_user
+from flask_wtf import FlaskForm
+from wtforms import StringField, TextAreaField
+from wtforms.validators import InputRequired
+
+from app.helpers.form_helper import is_valid_form
+from app.helpers import template_helper
+from app.submitter.encrypter import encrypt
+from app.submitter.submission_failed import SubmissionFailedException
+from app.globals import get_metadata
+from app.submitter.converter import convert_feedback
+from app.utilities.schema import load_schema_from_metadata
+
+from structlog import get_logger
+
+logger = get_logger()
+
+feedback_blueprint = Blueprint(name='feedback',
+                               import_name=__name__,
+                               url_prefix='/feedback')
+
+
+class FeedbackForm(FlaskForm):
+    message = TextAreaField('message', validators=[InputRequired()])
+    name = StringField('name')
+    email = StringField('email')
+
+
+@feedback_blueprint.before_request
+def before_request():
+    metadata = get_metadata(current_user)
+    if metadata:
+        logger.bind(tx_id=metadata['tx_id'])
+        g.schema_json = load_schema_from_metadata(metadata)
+
+
+@feedback_blueprint.route('', methods=['GET'])
+@login_required
+def get_form():
+    form = FeedbackForm()
+    return _render_template(
+        'feedback',
+        page_title='Survey feedback',
```

Comment:
The title text should be set in the template, should be settable via `{% block page_title %}Survey feedback{% endblock %}`